static/js/components/widgets/MarkdownEditor.tsx

```diff
@@ -103,11 +113,12 @@ export default function MarkdownEditor(props: Props): JSX.Element {
         onReady={setEditorRef}
         onChange={onChangeCB}
       />
-      {attach && attach.length > 0 ? (
+      {hasAttach ? (
         <ResourcePickerDialog
           open={resourcePickerOpen}
           setOpen={setResourcePickerOpen}
           insertEmbed={addResourceEmbed}
+          // @ts-ignore
```

Comment:
I think you can get rid of the `@ts-ignore` here if you inline the value of `hasAttach` (`attach && attach.length > 0`) - I think typescript is just not quite smart enough to figure that one out

you could also probably do an `attach as string` type coercion